package.json

```diff
@@ -21,9 +21,10 @@
   },
   "homepage": "https://github.com/northcoders/be-nc-games#readme",
   "dependencies": {
-    "pg-format": "^1.0.4",
     "dotenv": "^16.0.0",
-    "pg": "^8.7.3"
+    "pg": "^8.7.3",
+    "pg-format": "^1.0.4",
+    "supertest": "^6.3.0"
```

Comment:
- Supertest can be installed as a devDependency ... do you need Jest too?

Reply:
I have installed both supertest and Jest as dev dependencies now